### PR TITLE
Patch Tuesday June 2023 + Ubuntu 22.04

### PR DESCRIPTION
**Pull request by BillyONeal**
Azure Pipelines fixed Ubuntu 22.04 so we can finally adopt it.

Related:

braft https://github.com/microsoft/vcpkg/pull/31965
chipmunk https://github.com/microsoft/vcpkg/pull/31966
dbow3 https://github.com/microsoft/vcpkg/pull/31967
gppanel https://github.com/microsoft/vcpkg/pull/31968
hareflow https://github.com/microsoft/vcpkg/pull/31969
libbf https://github.com/microsoft/vcpkg/pull/31970
live555 https://github.com/microsoft/vcpkg/pull/31972
mathgl https://github.com/microsoft/vcpkg/pull/31976
mdl-sdk: https://github.com/microsoft/vcpkg/pull/32008
minisat-master-keying https://github.com/microsoft/vcpkg/pull/31977
nifly https://github.com/microsoft/vcpkg/pull/31998
sltbench https://github.com/microsoft/vcpkg/pull/31999
thor https://github.com/microsoft/vcpkg/pull/32001
unicorn-lib https://github.com/microsoft/vcpkg/pull/32002

ci.baseline.txt changes:

Need Python2
blitz
duktape
keystone

bond was just reordered together.

coin-or-ipopt fails to find lapack despite depending on intel-mkl; previously blocked by coinutils failing.

Have dynamic exception specs that are now rejected:
libodb
log4cpp

wpilib triggers what looks like a strange gcc-11 bug? Upstream has diverged substantially and it might make sense to remove the port entirely.

### CODE REVIEW

File: scripts/test_ports/vcpkg-find-acquire-program/portfile.cmake [hunk 1]
```diff
@@ -12,11 +12,10 @@ vcpkg_find_acquire_program(GN)
 vcpkg_find_acquire_program(NINJA)
 vcpkg_find_acquire_program(PERL)
 vcpkg_find_acquire_program(PKGCONFIG)
-vcpkg_find_acquire_program(PYTHON2)
 vcpkg_find_acquire_program(PYTHON3)
 vcpkg_find_acquire_program(YASM)
 
-if(NOT VCPKG_TARGET_IS_OSX)
+if(VCPKG_TARGET_IS_OSX)
```
Comment (dg0yt) [inline scripts/test_ports/vcpkg-find-acquire-program/portfile.cmake:18]: OSX was excluded. Shall it be the only platform for this test (MESON) now?

Comment (BillyONeal) [reply scripts/test_ports/vcpkg-find-acquire-program/portfile.cmake:18]: Oops, this is indeed bogus; I was going to insert an else but after consulting ci.baseline.txt ended up not doing that...

**Review by vicroms — DISMISSED** [review]

**Comment by BillyONeal** [comment]
dcmtk matches last baseline run.

openmvg was previously blocked by coin stuff:

```
Building openmvg[core,software]:x64-linux...
error: building openmvg:x64-linux failed with: CASCADED_DUE_TO_MISSING_DEPENDENCIES
    due to the following missing dependencies:
        coin-or-clp[core]:x64-linux
        coin-or-osi[core]:x64-linux
        coinutils[core]:x64-linux
Elapsed time to handle openmvg:x64-linux: 153 us
```

File: scripts/test_ports/vcpkg-find-acquire-program/portfile.cmake (hunk 1 above)
Comment (xXMoneyXx) [reply scripts/test_ports/vcpkg-find-acquire-program/portfile.cmake]: Crap

**Review by xXMoneyXx — COMMENTED** [review]
Up

**Review by xXMoneyXx — COMMENTED** [review]
Good

**Pull request merged**